langchain/document_loaders/trello.py

```diff
@@ -0,0 +1,124 @@
+"""Loader that loads cards from Trello"""
+from typing import List, Optional
+
+from langchain.docstore.document import Document
+from langchain.document_loaders.base import BaseLoader
+
+class TrelloLoader(BaseLoader):
+    """Trello loader. Reads all cards from a Trello board.
+
+    Args:
+        api_key (str): Trello API key.
+        api_token (str): Trello API token.
+    """
+
+    def __init__(self, api_key: str, api_token: str):
+        """Initialize Trello loader."""
+        self.api_key = api_key
+        self.api_token = api_token
+
+    def load(
+        self,
+        board_name: str,
+        card_filter: Optional[str] = "all",
+        include_card_name: Optional[bool] = True,
```

Comment:
Bools shouldn't have an optional since we only care about `True` and `False`, not about a `None` value. (At least in this case)

```suggestion
        include_card_name: bool = True,
```